tests/queries/0_stateless/02680_mysql_ast_logical_err.sql

```diff
@@ -1,4 +1,10 @@
 CREATE TABLE foo (key UInt32, a String, b Int64, c String) ENGINE = TinyLog;
 
-SELECT count() FROM mysql(mysql('127.0.0.1:9004', currentDatabase(), 'foo', 'default', ''), '127.0.0.1:9004', currentDatabase(), 'foo', '', ''); -- { serverError UNKNOWN_FUNCTION }
-SELECT count() FROM mysql(mysql('127.0.0.1:9004', currentDatabase(), 'foo', 'default', '', SETTINGS connection_pool_size = 1), '127.0.0.1:9004', currentDatabase(), 'foo', '', ''); -- { serverError UNKNOWN_FUNCTION, UNSUPPORTED_METHOD }
+SELECT count() FROM mysql(
+    mysql('127.0.0.1:9004', currentDatabase(), 'foo', 'default', ''),
+    '127.0.0.1:9004', currentDatabase(), 'foo', '', '',
+    SETTINGS connect_timeout = 100, connection_wait_timeout = 100, read_write_timeout = 300); -- { serverError UNKNOWN_FUNCTION }
+SELECT count() FROM mysql(
+    mysql('127.0.0.1:9004', currentDatabase(), 'foo', 'default', '', SETTINGS connection_pool_size = 1),
+    '127.0.0.1:9004', currentDatabase(), 'foo', '', '',
+    SETTINGS connect_timeout = 100, connection_wait_timeout = 100, read_write_timeout = 300); -- { serverError UNKNOWN_FUNCTION, UNSUPPORTED_METHOD }
```

Comment:
@alesapin I think the root cause is explained [here](https://github.com/ClickHouse/ClickHouse/pull/65938) due to misuse of millisecond and second. Through that fix, this issue should be fixed as well. A similar issue is [2479](https://github.com/ClickHouse/ClickHouse/issues/50911) with the same [stacktrace](https://pastila.nl/?004a11f2/524594607042611405be9dcca1006193#XEdv54r2kkgfb7vECQf/Xg==). The issue doesn't [appear](https://play.clickhouse.com/play?user=play#U0VMRUNUIHB1bGxfcmVxdWVzdF9udW1iZXIsIGNoZWNrX3N0YXJ0X3RpbWUsIGNoZWNrX25hbWUsIHRlc3RfbmFtZSwgdGVzdF9zdGF0dXMsIGNoZWNrX3N0YXR1cywgcmVwb3J0X3VybApGUk9NIGNoZWNrcwpXSEVSRSAxCiAgICBBTkQgcHVsbF9yZXF1ZXN0X251bWJlciA9IDAKICAgIEFORCB0ZXN0X3N0YXR1cyAhPSAnU0tJUFBFRCcKICAgIEFORCB0ZXN0X3N0YXR1cyAhPSAnT0snCiAgICBBTkQgY2hlY2tfc3RhdHVzICE9ICdzdWNjZXNzJwogICAgQU5EIHRlc3RfbmFtZSBsaWtlICclMDI2ODBfbXlzcWxfYXN0X2xvZ2ljYWxfZXJyJScKT1JERVIgQlkgY2hlY2tfc3RhcnRfdGltZSBkZXNjLCBjaGVja19uYW1lLCB0ZXN0X25hbWU=) after the fix. Setting parameters is just in case of any future change.